### DISCUSSION TITLE
Ventanas de revisión del examen parcial

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-PRG1#586] [Announcements]
Las iré publicando por aquí.

La primera: hoy miércoles 13, de 16:15h a 16:45h en mi despacho.

**Comment by beatriizorozco** [comment]
Hoy hay revisión?

**Comment by mmasias** [comment]
Sí, ayer estuve por la tarde. Y hoy viernes también. Si por alguna razón en
la hora de la ventana de revisión no me encuentran en mi despacho, por
favor probar pasados unos minutos.
-
Manuel Masías
http://es.linkedin.com/in/mmasias
